nativescript-angular/router/page-router-outlet.ts

```diff
@@ -229,7 +235,8 @@ export class PageRouterOutlet implements OnDestroy { // tslint:disable-line:dire
         resolver: ComponentFactoryResolver | null): void {
 
         if (this.locationStrategy._isPageNavigatingBack()) {
-            throw new Error("Currently in page back navigation - component should be reattached instead of activated.");
+            log("Currently in page back navigation - component should be reattached instead of activated. " + routeTransitionWarning);
+            return;
```

Comment:
@vakrilov This is actually the biggest culprit to Android crashes and app become *unresponsive* of them all. On Android only (very rarely if ever happens on iOS) sometimes the back page navigation transition timing is different and this could be called before `_finishBackPageNavigation` gets fired on location strategy however just logging and allowing it to finish will correct itself on it's own. Definitely don't want to `throw` here.